google-cloud-core/src/test/java/com/google/cloud/TimestampTest.java

```diff
@@ -257,6 +257,16 @@ public void parseTimestampWithoutTimeZoneOffset() {
         .isEqualTo(Timestamp.ofTimeSecondsAndNanos(TEST_TIME_SECONDS, 0));
   }
 
+  @Test
+  public void parseTimestampWithTimeZoneOffset() {
+    assertThat(Timestamp.parseTimestamp("0001-01-01T00:00:00-00:00"))
+        .isEqualTo(Timestamp.MIN_VALUE);
+    assertThat(Timestamp.parseTimestamp("9999-12-31T23:59:59.999999999-00:00"))
+        .isEqualTo(Timestamp.MAX_VALUE);
+    assertThat(Timestamp.parseTimestamp("2020-12-06T19:21:12.123+05:30")).isNotNull();
+    assertThat(Timestamp.parseTimestamp("2020-07-10T14:03:00-07:00")).isNotNull();
+  }
```

Comment:
the test will be stronger if replace isNotNull with isEqualTo(...)

Reply:
done